kusoCode.rb

```diff
@@ -0,0 +1,5 @@
+def badName
+  if something
```

Comment:
Style/IfUnlessModifier: Favor modifier `if` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`.